src/app/builders/components/Metrics.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+export const Metrics = () => {
+  return (
+    <div className="grid grid-cols-3 gap-8 w-full">
```

Comment:
Here I would also reuse the metrics component or create a new re-usable component.